docs/docs/setup-vast/install/docker.md

```diff
@@ -0,0 +1,42 @@
+# Docker
+
+:::tip Docker Hub
+We provide pre-built Docker images at [dockerhub.com/tenzir][dockerhub].
+:::
+
+[dockerhub]: https://hub.docker.com/repository/docker/tenzir/vast
+
+Our Docker image contains a dynamic of VAST build with plugins as shared
+libraries. The system user `vast` runs the VAST executable with limited
+privileges. Database contents go into the volume exposed at `/var/lib/vast`.
+
+## Start the container
+
+Start VAST in a container and detach it to the background.
+
+```bash
+mkdir -p /var/lib/vast
+docker run -dt --name=vast --rm -p 42000:42000 -v /var/lib/vast:/var/lib/vast \
+  tenzir/vast:latest start
+```
+
+## Build the main image
+
+Build the `tenzir/vast` image as follows:
+
+```bash
+docker build -t tenzir/vast:<TAG>
```

Comment:
@dominiklohmann is `TAG` optional?

Reply:
It defaults to `latest`. We upload this image though, not sure why we're providing build instructions here. The user should only be interested in _pulling_ the image.

Reply:
I've revamped the Docker docs. Mind giving it a read?